.github/workflows/tests.yml

```diff
@@ -9,10 +9,12 @@ env:
 jobs:
   tests:
     if: github.repository == 'Homebrew/brew'
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.config.os }}
```

Comment:
Is there any way to do this that preserves `matrix.os`? This was seems likely to be error-prone given the inconsistency with other jobs for the benefit of a single string title.

Reply:
I don't know of a way to do that... I'll revert this change.

Reply:
Shame, thanks 😭

Reply:
I found a way to do that :)